.mergify.yml

```diff
@@ -21,11 +13,37 @@ pull_request_rules:
       - -merged
       - -closed
       - author!=dependabot[bot]
-    actions:
-      queue:
-        method: squash
-        name: default
+    merge_conditions:
```

Comment:
`queue_rules->merge_conditions` replaces the old`queue_rules` "default" `conditions` logic